LetSwift/Helpers/Extensions/UICollectionViewExtension.swift

```diff
@@ -81,11 +81,11 @@ extension UICollectionView: NibPresentable {
         return ObservableEvent(event: delegateProxy.scrollViewWillBeginDraggingObservable)
     }
 
-    func registerNib(_ nib: UINib?, forCellReuseIdentifier identifier: String) {
+    @nonobjc func register(_ nib: UINib?, forCellReuseIdentifier identifier: String) {
```

Comment:
`@nonobjc` needed because both methods have the same Objective-C selector which is forbidden